docs/sampling/README.md

```diff
@@ -0,0 +1,282 @@
+For more information on how to perform and interpret probability
+sampling based on these properties, [consult OTEP 235][OTEP235].
+
+### Overriding sampling decisions
+
+Instrumentation authors and end-users that wish to prioritize an item
+of telemetry for collection in spite of sampling can add a
+`sampling.priority` attribute.  This attribute is a suggestion, a way
+of recognizing the importance of a certain event and requesting
+additional consideration from the collection pipeline.  For example, a
+user could write this code snippet:
+
+```
+   if err := doSomething(); err != nil {
+     if err == VERY_SERIOUS {
+       span.SetAttribute("sampling.priority", 10000)
+     }
+     return err
+   }
+```
+
+Samplers and sampling processors SHOULD pass items of telemetry to the
+exporter, independent of their default sampling mechanism, when the
+`sampling.priority` attribute is present and non-zero.  Although the
+priority is an integer, mathematical weight is not prescribed and no
+other specific behaviors are required.
+
+### Overriding sampling randomness
+
+Sampling system designers are able to override sampling randomness on
+an item-by-item basis, which may be done for several reasons, including
+situations where there is no TraceID defined.
+
+When a tracing system purposely uses TraceIDs that do not follow the
+W3C Trace Context Level 2 specification for TraceID randomness, and
+they wish to use OpenTelemetry sampling components, they can insert
+explicit randomness to prevent erroneously taking randomness from the
+TraceID.
+
+Another use for overriding sampling randomness is to configure a
+different unit of sampling consistency.  For example, multiple traces
+can be given the same randomness value to ensure that either all or
+none of them are sampled consistently.
+
+In the tracing signal, sampling randomness can be overridden by
+setting an R-value in the tracestate.  In the logging signal, sampling
+randomness can be overridden by setting the `sampling.randomness`
+attribute.
+
+### Sampling threshold
+
+When determining the Threshold value from an item of telemetry,
+sampler implementations SHOULD:
+
+- use the OpenTelemetry T-value field (`th`) in `tracestate` (spans only)
+- use the `sampling.threshold` attribute value, if present in the record attributes (logs only)
+
+In both cases, the Threshold value is represented by one to 14
+hexadecimal digits, allowing the use of variable-precision sampling
+probability.  When fewer than 14 digits are input, the string is
+padded with trailing zeros to make the correct number of bits (56).
+
+The zero Threshold value (encoded by a single `0`) corresponds with
+100% sampling.
+
+When Threshold is not provided, no information about probability
+sampling is available.
+
+### Sampling randomness
+
+When determining the Randomness value from an item of telemetry,
+sampler implementations SHOULD:
+
+- use the `tracestate` OpenTelemetry R-value field (`rv`) if it is present (spans only), or
```

Comment:
tracestate maybe present and accessible within the process through current span and I assume it's desirable for spans and nested logs to have the same value, should there be any coordination between tracestate from span context and the attribute value in this case?

Reply:
Agreed -- see 79275a3.